### PR TITLE
Localization for kernel-messaging example

### PR DESCRIPTION
**Pull request by kgoo124**
See #111

### CODE REVIEW

File: advanced/kernel-messaging/src/index.ts [hunk 1]
```diff
@@ -27,27 +29,30 @@ const extension: JupyterFrontEndPlugin<void> = {
...
  * @param launcher [optional] Jupyter Launcher
+ * @param translator Jupyter Translator
  */
 function activate(
   app: JupyterFrontEnd,
   palette: ICommandPalette,
   mainMenu: IMainMenu,
-  launcher: ILauncher | null
+  launcher: ILauncher | null,
+  translator: ITranslator
```
Comment (goanpeca) [inline advanced/kernel-messaging/src/index.ts:50]: Maybe we could switch this 2 to make it more readable and have "Optional" items at the end?

What do you think @kgoo124 ?

**Review by goanpeca — COMMENTED** [review]
Minor comment. Otherwise looks good!

Thanks for working on this @kgoo124 :-)

File: advanced/kernel-messaging/src/index.ts (hunk 1 above)
Comment (kgoo124) [reply advanced/kernel-messaging/src/index.ts]: Yes, that works! Fixed

**Comment by kgoo124** [comment]
@jtpio ready for merge

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**